lib/sparse/src/index/inverted_index/inverted_index_compressed_mmap.rs

```diff
@@ -29,14 +30,22 @@ use crate::index::posting_list_common::GenericPostingElement;
 const POSTING_HEADER_SIZE: usize = size_of::<PostingListFileHeader>();
 const INDEX_CONFIG_FILE_NAME: &str = "inverted_index_config.json";
 
+pub struct Version;
+
+impl StorageVersion for Version {
+    fn current_raw() -> &'static str {
+        "0.2.0"
```

Comment:
Do we manually define a version number here on purpose?

Reply:
Yes. There are two versions defined in this PR: `0.1.0` for the current uncompressed format and `0.2.0` for the new.

Previously I've used the crate version as the format version (like how it is done in other parts of the codebase). But since we're going to merge support for two format versions in a single crate, it's not convenient. Assuming that there is a single version is messy and blocks me from further PRs on this feature.

Reply:
As I remember from @xzfc explanation, he cannot use the crate version as a version here because he has to define both current and previous and the crate allows to set only one version. Am I right, @xzfc ?

Reply:
We can set initial version as any value. Can we set the last version the same as the crate version? And hardcode only the obsolete one?